<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_Interface</class>
 <widget class="QWidget" name="Test_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>349</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>737</width>
    <height>349</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="summaryTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="summaryButton">
       <property name="text">
        <string>Summarize</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="TalentTree" name="talentTree" native="true">
     <property name="minimumSize">
      <size>
       <width>431</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TalentTree</class>
   <extends>QWidget</extends>
   <header>widgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
